src/test/java/org/fundacionjala/virtualassistant/controllers/SpeechControllerTest.java

```diff
@@ -0,0 +1,56 @@
+package org.fundacionjala.virtualassistant.controllers;
+
+import org.fundacionjala.virtualassistant.service.SpeechService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.multipart.MultipartFile;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
please import specific class names instead of using org.junit.jupiter.api.Assertions.*; in the code